concordium-contracts-common-derive/Cargo.toml

```diff
@@ -2,16 +2,14 @@
 name = "concordium-contracts-common-derive"
 authors = ["Concordium <developers@concordium.com>"]
 version = "1.0.1"
```

Comment:
We should bump the version up so we do a new release of it.

Technically this is a breaking change so we need a new major.